specification/electronic-prescription-service-api.yaml

```diff
@@ -18,12 +18,11 @@ info:
     ### As a prescriber
     You can:
     * encode prescription data
-    * create a secondary care (outpatient) prescription
-    * cancel a secondary care (outpatient) prescription
+    * create and cancel a secondary care (outpatient) prescription
+    * create and cancel a primary care prescription
 
     You cannot currently use this API to:
-    * create a primary care prescription
-    * create a secondary care (other than outpatient) prescription
+    * create a or cancel secondary care (homecare) prescription
```

Comment:
is this supposed to be `and/or` and if so, would it be the same above under `You can:` ?

Reply:
yeh it was, I will update